target/linux/mvebu/image/solidrun-clearfog-gt-8k.bootscript

```diff
@@ -0,0 +1,10 @@
+setenv bootargs "root=PARTUUID=@ROOT@-02 rw rootwait"
```

Comment:
This boot script can be removed if You only want to support booting from MMC, there is already generic one provided with `Default-arm64`. But as I can see from images provided by SolidRun https://images.solid-build.xyz/8040/, default bootloader also provides distro boot variables (https://gitlab.denx.de/u-boot/u-boot/raw/master/doc/README.distro) so booting from USB should be possible with altering `load` commands a bit, which examples I'll show below.